nss-day-cohort-41/tabloidmvc-the-gefilte-fish
# View post comments

# Description
- User is directed to comments list view when 'View Comments' button is clicked in specific post
- Comments are sorted by latest creation date/time with most recent appearing at the top
- Title of the post is displayed at the top of the page
- User is re-directed back to the post details view when 'Back to post' button is clicked
Ticket #19 : https://github.com/nss-day-cohort-41/tabloidmvc-the-gefilte-fish/issues/19
## Type of change
Please delete options that are not relevant.
# Testing Instructions
- Pull down branch and CD to branch files
- Run updated SQL 1.Create_DB and 2.Seed_Data scripts to add sample data in Comments table
- Run app with debugger or dotnet run
- Once logged-in navigate to Post link in navbar and select View on any available Post
- At bottom of Post details view, click 'View Comments' button, this should direct the user to the comments list view
- Comments should be displayed along with button to go back to post details
# Checklist:
